### DISCUSSION TITLE
Importing SCORM hangs at 0%

### DISCUSSION THREAD

**Discussion by romanaegis** [chamilo/chamilo-lms#5220] [General]
I have added a learning path and am trying to import a SCORM. Both Chamilo and the Server claim they are set up properly. Permissions appear correct, according to both. I have deleted and reinstalled Chamilo, Changed its directory, renamed the file I am trying to upload, and addressed the software sternly.

**Comment by romanaegis** [comment]
It is not. It is on a website on a Linux server.

**Comment by romanaegis** [comment]
I made a small 1.3 MB SCORM and tried to upload that. It also hung at 0%.

**Comment by ywarnier** [comment]
If it hangs at 0% it doesn't seem like it would be related to the SCORM format itself. Rather related to the upload process. Can you upload the SCORM as a simple document in the documents tool?

**Comment by romanaegis** [comment]
I will try that. Do you know where the SCORM should go in the DB by any
chance?

**Reply by LudiscapeXApi** [reply]
The SCORM content goes into the Courses/SCORM folder and creates a record in the c_lp table
For 0% check the writing rights of upload php files and destination folders

**Comment by romanaegis** [comment]
Thanks, I'll check it out.

**Reply by Heleol** [reply]
did you manage to solve it? I have the same problem on a shared server with cpanel, Thanks
